lighthouse-core/config/default.js

```diff
@@ -241,6 +243,10 @@ module.exports = {
       title: 'Crawling and Indexing',
       description: 'To appear in search results, crawlers need access to your app.',
     },
+    'manual-seo-checks': {
+      title: 'Manual SEO Validation',
```

Comment:
Both accessibility and PWA categories call this subcategory 'Additional items to manually check'  - IMO we should have the same name here.

Reply:
+1 :)

Reply:
👍